core/src/test/scala/org/apache/spark/storage/BlockManagerSuite.scala

```diff
@@ -1074,7 +1074,7 @@ class BlockManagerSuite extends SparkFunSuite with Matchers with BeforeAndAfterE
     assert(memoryStore.currentUnrollMemoryForThisTask === 0)
 
     // Unroll with all the space in the world. This should succeed.
-    var putResult = memoryStore.putIterator("unroll", smallList.iterator, StorageLevel.MEMORY_ONLY)
+    var putResult = memoryStore.putIteratorAsValues("unroll", smallList.iterator)
```

Comment:
Good call; yes. I'll add some, since there's actually a couple of new branches that need coverage.